elyra/pipeline/catalog_connector.py

```diff
@@ -101,6 +154,8 @@ def read_catalog_entry(self,
                            catalog_entry_data: Dict[str, Any],
                            catalog_metadata: Dict[str, Any]) -> Optional[str]:
         """
+        DEPRECATED. Will be removed in 4.0. get_component_definition() must be implemented instead.
```

Comment:
Can we use the [`deprecated` decorator](https://deprecation.readthedocs.io/en/latest/) so this shows up in builds?